templates/rbac.yaml

```diff
@@ -15,13 +22,29 @@ metadata:
   name: coder
   namespace: {{ .Release.Namespace | quote }}
   annotations: {{ toYaml .Values.coderd.builtinProviderServiceAccount.annotations | nindent 4 }}
-  labels: {{ toYaml .Values.coderd.builtinProviderServiceAccount.labels | nindent 4 }}
+  labels:
+    app.kubernetes.io/name: {{ .Chart.Name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: {{ include "coder.serviceName" . }}
+    {{- with .Values.coderd.builtinProviderServiceAccount.labels -}}
```

Comment:
this behavior is similar for extraLabels on deployments, but unfortunately this name can't be changed without breaking backward compatibility